composer.json

```diff
@@ -12,15 +12,15 @@
     "require": {
         "composer/installers": "^2.1",
         "cweagans/composer-patches": "^1.7",
-        "drupal/core-composer-scaffold": "^11@beta",
-        "drupal/core-recommended": "^11@beta",
+        "drupal/core-composer-scaffold": "^11@alpha",
```

Comment:
this may seem like a "step back", but it actually makes things less confusing: the versions of `drupal/core-*` packages are inline with the `minimum-stability` currently set to `alpha`.

Reply:
@AlexSkrypnyk – We will finally change it to stable when stable is out, right?

Reply:
for sure :)

Please see a screenshot of the table from description for reference
<img width="378" alt="Cursor_and_Added_site_information_to_CI_and_`alpha`_job__by_AlexSkrypnyk_·_Pull_Request__671_·_drupal-composer_drupal-project" src="https://github.com/drupal-composer/drupal-project/assets/378794/598c8719-5b52-4c68-b49d-8ed83012dc7e">

Reply:
Alright!